components/aggregation/mat/StackedBarChart.js

```diff
@@ -22,15 +39,15 @@ export const StackedBarChart = ({ data, query }) => {
       let indexBy = ''
       indexBy = query['axis_x']
       let reshapedData = Array.isArray(data.data.result) ? data.data.result.map(d => {
-        d['ok_count'] = d.measurement_count - d.confirmed_count - d.anomaly_count
+        d['ok_count'] = d.measurement_count - d.anomaly_count - d.failure_count
         return d
       }) : data.data.result
       return {
         data: reshapedData,
         dimensionCount: data.data.dimension_count,
         url: data.url,
         loadTime: data.loadTime,
-        cols,
+        cols: (indexBy === 'blocking_type') ? ['measurement_count'] : cols,
```

Comment:
I'm not sure what this does. Can you clarify? I think we should for the time being omit logic for `blocking_type` plotting, because the data is not being formatted properly at the source and I don't think we are going to be able to properly display it.
What we really want to be doing with `blocking_type` is display it as values inside of each bar, not adding it as a new axis.

Reply:
I have documented the API work to be done for this here: https://github.com/ooni/backend/issues/554

Reply:
> I'm not sure what this does. Can you clarify?

Reply:
>I'm not sure what this does. Can you clarify?

For the 1-dimension chart, in case of `blocking_type` as X-axis, this replaces the stack with a simple bar showing the value in `measurement_count`. That was my interpretation of the data in the [API response](https://backend-fsn.ooni.org/api/v1/aggregation?probe_cc=AF&test_name=web_connectivity&since=2022-01-26&until=2022-01-28&axis_x=blocking_type).

Anyway, as discussed above, I will be removing `blocking_type` as an option for X-axis. So this will also be removed.